pkg/cli/admin/upgrade/status/controlplane.go

```diff
@@ -427,23 +434,53 @@ func vagueUnder(actual, estimated time.Duration) string {
 	}
 }
 
+func commaJoin(elems []UpdatingClusterOperator) string {
+	var names []string
+	for _, e := range elems {
+		names = append(names, e.Name)
+	}
+	return strings.Join(names, ", ")
+}
+
 var controlPlaneStatusTemplate = template.Must(
 	template.New("controlPlaneStatus").
-		Funcs(template.FuncMap{"shortDuration": shortDuration, "vagueUnder": vagueUnder}).
+		Funcs(template.FuncMap{"shortDuration": shortDuration, "vagueUnder": vagueUnder, "commaJoin": commaJoin}).
 		Parse(controlPlaneStatusTemplateRaw))
 
-func (d *controlPlaneStatusDisplayData) Write(f io.Writer) error {
+func (d *controlPlaneStatusDisplayData) Write(f io.Writer, detailed bool, now time.Time) error {
 	if d.Operators.Updated == d.Operators.Total {
 		_, err := f.Write([]byte(fmt.Sprintf("= Control Plane =\nUpdate to %s successfully completed at %s (duration: %s)\n", d.TargetVersion.target, d.CompletionAt.UTC().Format(time.RFC3339), shortDuration(d.Duration))))
 		return err
 	}
-	return controlPlaneStatusTemplate.Execute(f, d)
+	if err := controlPlaneStatusTemplate.Execute(f, d); err != nil {
+		return err
+	}
+	if detailed && len(d.Operators.Updating) > 0 {
+		table := tabwriter.NewWriter(f, 0, 0, 3, ' ', 0)
+		f.Write([]byte("\nUpdating Cluster Operators"))
+		_, _ = table.Write([]byte("\nNAME\tSINCE\tREASON\tMESSAGE\n"))
+		for _, o := range d.Operators.Updating {
+			reason := o.Condition.Reason
+			if reason == "" {
+				reason = "-"
+			}
+			_, _ = table.Write([]byte(o.Name + "\t"))
+			_, _ = table.Write([]byte(shortDuration(now.Sub(o.Condition.LastTransitionTime.Time)) + "\t"))
+			_, _ = table.Write([]byte(reason + "\t"))
+			_, _ = table.Write([]byte(o.Condition.Message + "\n"))
+		}
+		if err := table.Flush(); err != nil {
+			return err
+		}
+	}
+	return nil
 }
 
 const controlPlaneStatusTemplateRaw = `= Control Plane =
 Assessment:      {{ .Assessment }}
 Target Version:  {{ .TargetVersion }}
-Completion:      {{ printf "%.0f" .Completion }}% ({{ .Operators.Updated }} operators updated, {{ .Operators.Updating }} updating, {{ .Operators.Waiting }} waiting)
+{{ if commaJoin .Operators.Updating }}Updating:        {{ commaJoin .Operators.Updating }}{{ end }}
```

Comment:
```suggestion
{{ if commaJoin .Operators.Updating -}}
Updating:        {{ commaJoin .Operators.Updating }}
{{ end -}}
```

Reply:
Actually, one more template trick to avoid double `commaJoin` call:

```
{{ with commaJoin .Operators.Updating -}}
Updating:        {{ . }}
{{ end -}}
```